transcom/mymove
# INT B-18316 remove rank & duty_location from service_member table

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3a873926)

## Summary

With the merge of [E-04823](https://github.com/transcom/mymove/pull/11893) which removed the use of `rank` and `duty_location` off of the service member and now uses data from the `orders` table, we can make database changes to reduce technical debt.

I made the appropriate database changes and navigated through several flows as both customer and TOO and did not encounter any errors, but would love a thorough check from the testers of this PR.

This PR does the following:
- removes `rank` column from `service_members` table
- removes `duty_location_id` column from `service_members` table

This was done with a migration file that contains the following:
```
ALTER TABLE service_members
DROP COLUMN rank,
DROP COLUMN duty_location_id;
```

### How to test

1. Access MM as customer & office
2. Navigate the application in all flows to ensure no errors occur (particularly in profile editing/viewing)

### Database
- removed `rank` from table `service_members`
- removed `duty_location_id` from `service_members`
